saleor/order/models.py

```diff
@@ -500,3 +500,10 @@ class Meta:
 
     def __repr__(self):
         return f"{self.__class__.__name__}(type={self.type!r}, user={self.user!r})"
+
+
+class Invoice(models.Model):
+    order = models.ForeignKey(Order, on_delete=models.CASCADE)
```

Comment:
should this be cascade?

Reply:
Good question. @korycins @patrys should we keep invoices in DB when you delete related orders? It could be useful to store them as historical data. It's not usual (and not possible through UI/API) to remove orders but still, maybe some integration could be able to do that at some point?

Reply:
We could make this nullable but I don't think we want to go beyond that for now (eg. no new UI to show abandoned invoices). I think it would be better to anonymize and flag orders instead of actually deleting them as we're also deleting their event logs.